### PR TITLE
deprecated timeline

### PR DESCRIPTION
**Pull request by dtailor90**
@larkin @charlesh88 @psarram - Please look at my first version of deprecated timelines, is this how we want it to look to users that have existing timelines? 

Note: To test, create a timeline using another branch, and switch to this branch to see the deprecation warning.

### CODE REVIEW

**Comment by dtailor90** [comment]
@charlesh88 - Good call, I made the changes you requested.

File: platform/features/timeline/bundle.js [hunk 1]
```diff
@@ -83,9 +84,10 @@ define([
     legendItemTemplate,
     ticksTemplate,
     datetimeTemplate,
+    deprecatedTimelineMessage,
     legacyRegistry
 ) {
-
+    /*
```
Comment (larkin) [inline platform/features/timeline/bundle.js]: Just delete everything; don't comment it out.  We can always retrieve it later.

File: platform/features/timeline/bundle.js [hunk 2]
```diff
@@ -616,4 +618,28 @@ define([
...
+*/
+    // Timeline, Activity and Activity Mode objects have been deprecated and will no longer be supported, to use timelines, uncomment the code above and comment out the code below.
+
+    legacyRegistry.register('platform/features/timeline', {
+        extensions: {
+            types: [
+                {
+                    key: "timeline",
+                    name: "Timeline",
+                    description: "A time-oriented container that lets you enclose and organize other Timelines and Activities. The Timeline view provides both tabular and Gantt views as well as resource utilization graphing of Activities.",
```
Comment (larkin) [inline platform/features/timeline/bundle.js]: can this say the same thing as the deprecated timeline message?

**Comment by dtailor90** [comment]
## Author Checklist

1. Changes address original issue? Y
2. Unit tests included and/or updated with changes? Removed Tests corresponding to timelines.
3. Command line build passes? Y
4. Changes have been smoke-tested? Y

**Comment by larkin** [comment]
# Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? N/A
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y

**Pull request merged**